lib/membrane_raw_audio.ex

```diff
@@ -69,95 +68,95 @@ defmodule Membrane.RawAudio do
   end
 
   @doc """
-  Determines if format is floating point.
+  Determines if sample format is floating point.
```

Comment:
I guess it's a good opportunity to improve the docs here :wink: 
```suggestion
  Determines if the sample values are represented by a floating point number.
```

Reply:
You're right but that's not only this function... I guess I can take care of it